ir_api/core/specifications/run.py

```diff
@@ -0,0 +1,54 @@
+"""
+Module defining specifications for querying Run entities in the IR API.
+
+It includes the RunSpecification class which allows for building complex queries
+for Run entities based on various criteria such as instrument, limit, offset,
+order by, and order direction.
+"""
+# pylint: disable=unused-argument
```

Comment:
See: https://github.com/interactivereduction/IR-API/pull/277/files#diff-c84d21f7021977eeefaca8726ed64293b40713a2ad0b5acfa6afc78fe7841072R64
for disable reason